atomic_reactor/plugins/exit_koji_promote.py

```diff
@@ -401,7 +404,7 @@ def add_log_type(output):
         repositories = [image.to_str() for image in output_images
                         if image.tag != 'latest']
         arch = os.uname()[4]
```

Comment:
Oh, right, that's true. And it already does provide the arch to osbs-client so it can fill in the architecture label.